lib/ceo/rails/routes.rb

```diff
@@ -19,6 +19,8 @@ def admin_for(*rsrcs, &block)
       options[:concerns] = :pageable
 
       namespace :admin do
+        get 'dashboard', to: 'admin#dashboard'
```

Comment:
You're completely right @JesseHerrick, thanks for pointing that out. I'll add that to my to-do list.